### PR TITLE
Migrate OhMySmtp to Mailpace

### PR DESCRIPTION
**Pull request by princemaple**

### CODE REVIEW

File: lib/swoosh/adapters/mail_place.ex [hunk 1]
```diff
@@ -0,0 +1,116 @@
+defmodule Swoosh.Adapters.MailPace do
```
Comment (taobojlen) [inline lib/swoosh/adapters/mail_place.ex:1]: This file is called `mail_place.ex` but should be `mail_pace.ex`.

Comment (princemaple) [reply lib/swoosh/adapters/mail_place.ex:1]: Yup, my bad

File: test/swoosh/adapters/oh_my_smtp_test.exs [hunk 2]
```diff
@@ -1,8 +1,8 @@
-defmodule Swoosh.Adapters.OhMySmtpTest do
+defmodule Swoosh.Adapters.MailPaceTest do
```
Comment (kurtome) [inline test/swoosh/adapters/oh_my_smtp_test.exs:1]: Looks like the file name should match the module name

Comment (princemaple) [reply test/swoosh/adapters/oh_my_smtp_test.exs:1]: haha yep.

**Comment by princemaple** [comment]
Thank you both for the review. @taobojlen @kurtome 

If not more changes are required I'll merge it in a couple hours and release a new version.

**Pull request merged**